src/com/epam/javacourse/hometask3/Book.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.javacourse.hometask3;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public final class Book extends Bookstore {
+    private final String isbn;
+    private final String name;
+    private final Author authors;
+    private final Publisher publisher;
+    private final String publicationDate;
+    private final boolean isPaperback;
+
+    public Book(String isbn, String name, Author authors, Publisher publisher, String publicationDate,
+                boolean isPaperback) {
+        this.isbn = isbn;
+        this.name = name;
+        this.authors = authors;
+        this.publisher = publisher;
+        this.publicationDate = publicationDate;
+        this.isPaperback = isPaperback;
+    }
+
+    public Author[] getAuthors() {
+        return new Author[]{authors};
+    }
+
+    @Override
+    public String toString() {
+        return "Book{" +
+                "isbn='" + isbn + '\'' +
+                ", name='" + name + '\'' +
+                ", authors=" + Arrays.toString(new Author[]{authors}) +
```

Comment:
If you change `Arrays.toString(new Author[]{authors})` to `authors` according to the comment on line 25, then you can delete `authors=` from this line, as soon as you already have it in `Author` `toString()` method.